client/landing/stepper/declarative-flow/internals/steps-repository/components/calendy-widget/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useMemo } from 'react';
+
+declare global {
+	interface Window {
+		Calendly: any;
+	}
+}
+
+interface PrefillData {
+	name?: string;
+	email?: string;
+	// Add other prefill fields as needed
+}
+
+interface CalendlyWidgetProps {
+	url: string;
+	id?: string;
+	prefill?: PrefillData;
+	onSchedule?: () => void;
+}
```

Comment:
How about moving this code block to the dedicated `types.ts` file?

Reply:
I don't know. I don't expect them to be reused, so I feel like they're at a better place close to the component itself.